rusk-uplink/src/definitions.rs

```diff
@@ -102,6 +102,10 @@ pub trait Transaction: Archive {
     type Return;
 }
 
+pub trait Event: Archive {
```

Comment:
Consider removing this trait, as discussed.

Reply:
Removed